### DISCUSSION TITLE
Please make virtual display work on Windows Server 2012 R2

### DISCUSSION THREAD

**Discussion by killitkid13** [rustdesk/rustdesk#9955] [General]
Please make virtual display work on Windows Server 2012 R2

**Comment by fufesou** [comment]
Hi, could please try if the follwing idds work?

1. https://github.com/rustdesk-org/rdev/releases/download/usbmmidd_v2/usbmmidd_v2.zip
2. https://github.com/VirtualDisplay/Virtual-Display-Driver

Please don't use the default mstsc(RDP) to connect. Mstsc(RDP) and idd will not work at the same time
